src/Microsoft.Extensions.Localization/ResourceManagerStringLocalizer.cs

```diff
@@ -153,6 +153,13 @@ protected IEnumerable<LocalizedString> GetAllStrings(bool includeParentCultures,
                 ? GetResourceNamesFromCultureHierarchy(culture)
                 : GetResourceNamesForCulture(culture);
 
+            if (resourceNames == null && !includeParentCultures)
+            {
+                var resourceStreamName = GetResourceStreamName(culture);
+                throw new MissingManifestResourceException(
+                            string.Format(Resources.Localization_MissingManifest, resourceStreamName));
```

Comment:
@pranavkm was talking about eventually doing away with it. What happens if you run `./build resx` at the root of the repo?